mlir/lib/Catalyst/Transforms/DetectQNodes.cpp

```diff
@@ -93,6 +94,17 @@ LLVM::LLVMFuncOp lookupOrCreateAwaitTokenName(ModuleOp);
 LLVM::LLVMFuncOp lookupOrCreateDropRef(ModuleOp);
 
 void collectValuesToLookFor(ResultRange &results, SmallVector<Value> &valuesToLookFor);
+void collectResultsForMlirAsyncRuntimeErrorFunctions(SmallVector<Value> &values,
+                                                     SmallVector<Value> &results);
+void collectPotentialConditions(SmallVector<Value> &values, SmallVector<Value> &conditions);
+void collectSuccessorBlocks(SmallVector<Value> &conditions, SmallVector<Block *> &aborts,
+                            SmallVector<Block *> &success);
+void collectCallsToAbortInBlocks(SmallVector<Block *> &blocks, SmallVector<LLVM::CallOp> &calls);
+void replaceCallsWithCallToTarget(SmallVector<LLVM::CallOp> &oldCallOps, LLVM::LLVMFuncOp target,
+                                  SmallVector<LLVM::CallOp> &newCalls, PatternRewriter &rewriter);
+void replaceTerminatorWithUnconditionalJumpToSuccessBlock(SmallVector<Block *> abortBlocks,
+                                                          SmallVector<Block *> successBlocks,
+                                                          PatternRewriter &rewriter);
```

Comment:
This set of functions were not declared here before. This does not seem to be a "Documentation" change. I would suggest making a different commit for those kind of changes.

Reply:
I am moving functions such that the documentation can read from top to bottom. In order to have the code compile, I just needed to add declaration for functions.